docs/src/whatsnew/latest.rst

```diff
@@ -66,13 +68,21 @@ This document explains the changes made to Iris for this release
 📚 Documentation
 ================
 
-#. N/A
+#. `@ESadek-MO`_ and `@trexfeathers`_ created :ref:`contributing_pytest_conversions`
+   as a guide for converting from ``unittest`` to ``pytest``. (:pull:`5785`)
+
+#. `@ESadek-MO`_ and `@trexfeathers`_ created a style guide for ``pytest`` tests,
+   and consolidated ``Test Categories`` and ``Testing Tools`` into
+   :ref:`contributing_pytest_conversions` (:issue:`5574`, :pull:`5787`)
 
 
 💼 Internal
 ===========
 
-#. N/A
+#. `@ESadek_MO`_ `@pp-mo`_ `@bjlittle`_ `@trexfeathers`_ and `@HGWright`_ have
+   converted around a third of Iris' ``unittest`` style tests to ``pytest``. This is
+   part of an ongoing effort to move from ``unittest`` to ``pytest``. (:pull:`6207`,
+   part of :issue:`5690`)
```

Comment:
Are you planning to use #5960 for the future tracking of PyTest tasks. I would be up for completely rewriting the body to be easier to read, so long as we preserve the code snippet somewhere.

If you're planning to raise a new issue then I would rather reference that.